Crowd Strike Falcon X - bug with uploading file (#11675)
demisto/content: Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/README.md

| proxy | Use system proxy settings | False |

4. Click **Test** to validate the URLs, token, and connection.

##Uploading a file to the sandbox

> Please pay attention that there are 2 ways to upload a file to the sandbox.

Reviewer: ```suggestion
There are 2 ways to upload a file to the sandbox.
```

> 1. Using ***cs-fx-upload-file*** command with **submit_file=yes**.

Reviewer: ```suggestion
1. Using the ***cs-fx-upload-file*** command with **submit_file=yes**.
```

> 2. Using ***cs-fx-upload-file*** command and afterwards the ***cs-fx-submit-uploaded-file command***, 

Reviewer: ```suggestion
2. Using the ***cs-fx-upload-file*** command and afterwards the ***cs-fx-submit-uploaded-file command***, 
```

in this option the sha256 identifier from the ***cs-fx-upload-file*** command output is the input to the ***cs-fx-submit-uploaded-file command***.

> For more information look at the commands documentation.

Reviewer: ```suggestion
For more information review the documentation for the commands.
```
